invenio_celery/ext.py

```diff
@@ -34,10 +34,18 @@ def __init__(self, app=None, **kwargs):
     def init_app(self, app, entry_point_group="invenio_celery.tasks", **kwargs):
         """Initialize application object."""
         self.init_config(app)
+        app.logger.setLevel("DEBUG")
         self.celery = FlaskCeleryExt(app).celery
         self.entry_point_group = entry_point_group
+        self.load_celery_signals()
         app.extensions["invenio-celery"] = self
 
+    def load_celery_signals(self):
+        """Load Celery signals."""
+        # Import Celery signals via entry points
+        for ep in pkg_resources.iter_entry_points("invenio_celery.signals"):
```

Comment:
I addded this to have a common place where the signals are registered, it could also be done in each repo if we prefer - instead of using an entry point